code/game/atoms_movable.dm

```diff
@@ -126,7 +126,7 @@
 		destination.Entered(src, oldloc)
 		var/area/old_area = get_area(oldloc)
 		var/area/destarea = get_area(destination)
-		if(old_area != destarea)
+		if(destarea && old_area != destarea)

```

Comment:
It looked like get_area could reasonably fail, we have other if checks for it in other places.